config/runtime/http.go

```diff
@@ -71,6 +73,7 @@ func NewHTTPConfig(c *config.Gateway, args command.Args) (*HTTPConfig, error) {
 	set.StringVar(&conf.LogFormat, "log-format", conf.LogFormat, "-log-format json")
 	set.IntVar(&conf.ListenPort, "p", conf.ListenPort, "-p 8080")
 	set.BoolVar(&conf.UseXFH, "xfh", conf.UseXFH, "-xfh")
+	set.StringVar(&conf.RequestIDFormat, "request-id-format", "common", "-request-id-format uuid4")
```

Comment:
Instead of `common` the provided `conf` should be used as default value to apply internal defaults or configuration file settings if no command line argument is given.

Reply:
So it should be `conf.RequestIDFormat`?

Reply:
yes